### ISSUE TITLE
ModernTCN kernel_size & stirde is missed up with downsampling_ratio parma

### ISSUE THREAD

**Issue by yoryory** [WenjieDu/PyPOTS#769]
### 1. System Info

thank you for your work!
when using ModernTCN  i noticed that 

            for i in range(self.num_stage - 1):
                downsample_layer = nn.Sequential(
                    nn.BatchNorm1d(dims[i]),
                    nn.Conv1d(
                        dims[i],
                        dims[i + 1],
                        kernel_size=downsampling_ratio,
                        stride=downsampling_ratio,
                    ),
                )
                self.downsample_layers.append(downsample_layer)

for my understanding these two params should not  downsampling_ratio

### 2. Information

- [ ] The official example scripts
- [ ] My own created scripts

### 3. Reproduction

.

### 4. Expected behavior

.

### 5. Your contribution

.

**Comment by WenjieDu** [comment]
@yoryory Sorry, I don't get your point. Maybe you should refer to the official implementation first https://github.com/luodhhh/ModernTCN/blob/56a9a2c018385cd5acef015378cae7f084d1b11c/ModernTCN-classification/models/ModernTCN.py#L227-L233
